api/provider/aws/load_balancer_read.go

```diff
@@ -31,9 +29,7 @@ func (l *LoadBalancerProvider) Read(loadBalancerID string) (*models.LoadBalancer
 
 		if certificateARN := aws.StringValue(description.Listener.SSLCertificateId); certificateARN != "" {
 			// certificate arn format:  arn:aws:iam:region:012345678910:certificate/path/to/name
```

Comment:
we don't need this comment anymore since its only purpose was to help explain the string manipulation logic